docs/_entities/comment.md

```diff
@@ -50,7 +48,6 @@ See also: [Relayable][relayable]
   <parent_guid>c3893bf029e7013487753131731751e9</parent_guid>
   <text>this is a very informative comment</text>
   <author_signature>cGIsxB5hU/94+rmgIg/Z+OUvXVYcY/kMOvc267ybpk1pT44P1JiWfnI26F1Mta62UjzIW/SjeAO0RIsJRguaISLpXX/d5DJCMpePAZaZiagUbdgH/w4L++fXiPxBKkSm+PB4txxmHGN8FHjwEUJFHJ1m3VfU4w2JC8+IBU93eag=</author_signature>
-  <parent_author_signature>uzjxUSqR8DQBSBa6abY7R/s9DVzT6UAgTctRcUu5rV5o0iXJD2MR6kp6bsVH3nMbbNvOjwAtrdfz3SVHT2gD8M5PmoFagxK7m5T1c9FB0i+wknoAah0Si0c2sP/BPLnkQ83DgLjF+JZCzMX4sWKvYfyaMjnih1MtQILGyuiwA0E=</parent_author_signature>
```

Comment:
You are right, thanks for the hint. I removed the separation between "From author" and "From parent author" and squashed it to the commit where I already removed `parent_author_signature` from the documentation.